service/route_handlers.go

```diff
@@ -448,15 +457,15 @@ func ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		HandleSamlAssertion(w, r, assertion, server.SamlServiceProvider)
+		HandleSamlAssertion(w, r, assertion, server.SamlServiceProvider, redirectBackBaseValue, redirectBackPathValue)
 		return
 	}
 
 	http.NotFoundHandler().ServeHTTP(w, r)
 }
 
-func getPossibleRequestIDs(r *http.Request, s *model.RancherSamlServiceProvider) []string {
-	rv := []string{}
+func getPossibleRequestIDs(r *http.Request, s *model.RancherSamlServiceProvider) (reqIDs []string, redirectBackBaseValue string,
```

Comment:
getPossibleRequestIDs does not accurately reflect what this function does

Reply:
why have you named the return parameters? My opinion, for clarity you shouldnt do that unless you really need to

Reply:
I thought I'll just name them since they contain an array of strings and 2 strings.

Reply:
@cjellick any reason from 'go' perspective we should not be naming the return fields? Atleast these made it easy for me to read the code.